release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -95,6 +95,10 @@ For more information about enabling a feature set during installation, see xref:
 [id="ocp-4-12-post-installation"]
 === Post-installation configuration
 
+==== {product-title} with multi-architecture compute machines (Technology Preview)
+
+{product-title} {product-version} with multi-architecture compute machines now supports manifest listed images on image streams. You can now use node affinity on the Operator Lifecycle Manager (OLM) to specify nodes of different architectures in your cluster. For more information, see xref:../post_installation_configuration/multi-architecture-configuration.adoc[Configuring multi-architecture compute machines on an {product-title} cluster]
```

Comment:
i think more appropriate would be - "you can now override nodeAffinity through the subscription object to schedule pods on nodes with the supported operator architecture"